proposals/20200818-artifacts-storage.md

```diff
@@ -0,0 +1,83 @@
+# Falco Artifacts Storage
+
+This document reflects the way we store the Falco artifacts.
+
+## Terms & Definitions
+
+- [Falco artifacts](./20200506-artifacts-scope-part-1.md)
+- Bintray: artifacts distribution platform
+
+## Packages
+
+The Falco packages are **automatically** built and sent to [bintray](https://bintray.com/falcosecurity) in the following cases:
+
+- a pull request gets merged into the master branch (**Falco development releases**)
```

Comment:
Discussing with @fntlnz and @leogr we also agreed that eventually we could stop publishing (not building, clearly) the Falco development releases every time a merge happens on the master branch.

This because we already have them downloadable from the CircleCI app for 30 days.

Please vote here.

I'll update the proposal accordingly.

Reply:
Cons:

- No more development repository for DEB, RPM, tarballs Falco packages.

Reply:
👍